.packit.yaml

```diff
@@ -0,0 +1,41 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+
+specfile_path: rubygem-foreman_theme_satellite.spec
+
+# add or remove files that should be synced
+files_to_sync:
+    - rubygem-foreman_theme_satellite.spec
+    - .packit.yaml
+
+# name in upstream package repository or registry (e.g. in PyPI)
+upstream_package_name: foreman_theme_satellite
+# downstream (Fedora) RPM package name
+downstream_package_name: rubygem-foreman_theme_satellite
+
+upstream_tag_template: v{version}
+
+actions:
+  post-upstream-clone:
+    #- "wget https://raw.githubusercontent.com/theforeman/foreman-packaging/rpm/develop/packages/satellite/rubygem-foreman_theme_satellite/rubygem-foreman_theme_satellite.spec -O rubygem-foreman_theme_satellite.spec"
+    - "wget https://raw.githubusercontent.com/evgeni/foreman-packaging/theme/packages/satellite/rubygem-foreman_theme_satellite/rubygem-foreman_theme_satellite.spec -O rubygem-foreman_theme_satellite.spec"
```

Comment:
https://github.com/theforeman/foreman-packaging/pull/9911

Reply:
this is now merged and updated